src/Network/Session.php

```diff
@@ -455,7 +455,7 @@ public function write($name, $value = null)
      */
     public function id($id = null)
     {
-        if ($id !== null && !headers_sent()) {
```

Comment:
This change was to fix issues with PHP7.2 and headers sent warnings.

Reply:
Hmm, so what alternative do you propose?
`$this->_isCLI || !headers_sent()` check?

Reply:
That should work.